### PR TITLE
Fixing Issue where removing Ingredients and Recipes wasn't added

### PR DESCRIPTION
**Pull request by MarshMello0**
It seems you forgot to remove the items from the `CraftbotDocument` list.

### CODE REVIEW

**Comment by grantlmul** [comment]
woah someone did a thing with a thing i made

File: RecipeEditor/Crafting.cs [hunk 1]
```diff
@@ -17,6 +17,12 @@ public void AddIngredient(String name)
         {
             this.ingredientList.Add(new Item(Form1.ItemDictionaryReversed[name]));
         }
+        public void RemoveIngredient(string name)
+        {
+            Item itemToRemove = ingredientList.Single(i => i.itemId == Form1.ItemDictionaryReversed[name]);
```
Comment (grantlmul) [inline RecipeEditor/Crafting.cs:22]: I had no clue what `=>` means until today. I'm pretty sure I still don't.

Comment (MarshMello0) [reply RecipeEditor/Crafting.cs:22]: It's just a simplified foreach loop with LINQ Queries.

Comment (grantlmul) [reply RecipeEditor/Crafting.cs:22]: Time to look up what LINQ does.

**Pull request merged**